### ISSUE TITLE
Error when building for iOS

### ISSUE THREAD

**Issue by jpmarr** [espresso3389/flutter_pdf_render#2]
Hi

Using this great on an Android device (on Windows) but when I try to run and deploy to the iOS Simulator on OSX, I'm getting the following error during the XCode build phase:

```
fatal error: 'pdf_render/pdf_render-Swift.h' file not found
#import <pdf_render/pdf_render-Swift.h>

```
Apologies - I'm not an iOS dev so I may be missing something trivially obvious.

**Comment by espresso3389** [comment]
Could you please let me know the result of `flutter doctor`?

**Comment by jpmarr** [comment]
Hi

```
[✓] Flutter (Channel stable, v1.7.8+hotfix.4, on Mac OS X 10.13.6 17G8030, locale en-GB)
    • Flutter version 1.7.8+hotfix.4 at /Users/Jim/Development/flutter
    • Framework revision 20e59316b8 (4 weeks ago), 2019-07-18 20:04:33 -0700
    • Engine revision fee001c93f
    • Dart version 2.4.0

 
[✓] Android toolchain - develop for Android devices (Android SDK version 29.0.2)
    • Android SDK at /Users/Jim/Library/Android/sdk
    • Android NDK location not configured (optional; useful for native profiling support)
    • Platform android-29, build-tools 29.0.2
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_152-release-1343-b01)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 10.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 10.1, Build version 10B61
    • CocoaPods version 1.7.5

[✓] iOS tools - develop for iOS devices
    • ios-deploy 1.9.4

[✓] Android Studio (version 3.4)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin version 38.2.1
    • Dart plugin version 183.6270
    • Java version OpenJDK Runtime Environment (build 1.8.0_152-release-1343-b01)

[✓] VS Code (version 1.37.0)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.3.0

[✓] Connected device (1 available)
    • iPhone XR • 72EDA4B6-08BA-4090-8B73-3AE40AAB8F38 • ios • iOS 12.1 (simulator)

• No issues found!
```

So - I've managed to get it working by adding:

`use_frameworks! 
`
to the Podfile.

However, that does make the project incompatible with firebase_messaging. I'll keep digging into that. Thanks for your response and a very useful component!

**Comment by espresso3389** [comment]
> So - I've managed to get it working by adding:
> 
> `use_frameworks! `

According to firebase_messaging's [CHANGELOG.md](https://github.com/flutter/plugins/blob/2d9cd26f29c0c2482b37e89b4be6500cdb70dd0a/packages/firebase_messaging/CHANGELOG.md#4004), the hack to remove `use_frameworks!` was fixed at least on 4.0.0+4.
